tests/index.js

```diff
@@ -19,36 +19,33 @@ testPdData.body = {};
 
 describe('index.js >', () => {
   describe(' createTTE tests', () => {
-    it('should return a single tte acknowledge_log_entry', () => {
+    it('should return a single tte with null as end time', () => {
```

Comment:
I would go with something more business domain like: should return end time null/empty/etc. when its not resolved